Makefile

```diff
@@ -706,6 +706,7 @@ SPEED_OPTIMISED_SRC := $(SPEED_OPTIMISED_SRC) \
             telemetry/esc_telemetry.c \
 
 SIZE_OPTIMISED_SRC := $(SIZE_OPTIMISED_SRC) \
+            fc/fc_msp.c \
```

Comment:
That is probably not a good idea. In general we try to only set code that isn't executed while armed to be optimised for size. `fc_msp.c` is used in-flight (and repetitively) by OSD and MSP over S.PORT. If we need to optimise for size, we can probably find better candidates than `fc_msp.c`.